model/logs/azure.yaml

```diff
@@ -0,0 +1,98 @@
+groups:
+  - id: az.resource.log
+    stability: experimental
+    type: event
+    name: az.log
+    brief: >
+      Describes Azure Resource Log event, see
+      [Azure Resource Log Top-level Schema](https://learn.microsoft.com/azure/azure-monitor/essentials/resource-logs-schema#top-level-common-schema)
+      for more details.
+    attributes:
+      - ref: event.name
+      - ref: az.service_request_id
+      - ref: cloud.resource_id
+    # Future Note: When the build tools support this definition please uncomment and validate the details
```

Comment:
nit: could you please indent the comment?

Reply:
The comment is for the body tag below so I added the comment to the same level. I think if I indented again it'll look like a comment to the above `ref:` entry. Is that what you mean?

Reply:
I mean GH warning above, I think GH yaml linter wants the comment to be indented on the same level as `ref`.  

I.e.
```yaml
- ref: cloud.resource_id
  # Future Note: When the build tools support this definition please uncomment and validate the details
```